Add a note about the known limitations of the credential claim metadata (#280)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

          * `locale`: OPTIONAL. String value that identifies language of this object represented as language tag values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
* `order`: OPTIONAL. Array of the claim name values that lists them in the order they should be displayed by the Wallet.

> The above metadata structure has some known limitations:
>
> * It cannot be used to describe claims in credentials that have the name `mandatory`, `value_type` or `display`.

> > * It is not possible to provide `mandatory`, `value_type` or `display` values for objects that contain claims 

Reviewer: this is the most important and should be the first one. and it should also be much more clearer.
```suggestion
> * It is not possible to provide `mandatory`, `value_type` or `display` values for the intermediary claims in the nested objects
```

> * The `order` parameter cannot be used for claims within objects
> * Arrays of unknown size cannot be described
>

> > These limitations are expected to be resolved in the second Implementer's Draft, a proposal can be viewed in [Issue 266](https://github.com/openid/OpenID4VCI/issues/266).

Reviewer: I am not sure about the current framing... I think the following is sufficient.
```suggestion
> Mechanisms that could address these limitations are being discussed in [Issue 266](https://github.com/openid/OpenID4VCI/issues/266).
```

> > The above metadata structure has some known limitations:
> >
> > * It cannot be used to describe claims in credentials that have the name `mandatory`, `value_type` or `display`.
> > * It is not possible to provide `mandatory`, `value_type` or `display` values for objects that contain claims 
> > * The `order` parameter cannot be used for claims within objects
> > * Arrays of unknown size cannot be described
> >
> > These limitations are expected to be resolved in the second Implementer's Draft, a proposal can be viewed in [Issue 266](https://github.com/openid/OpenID4VCI/issues/266).
> 

Reviewer: we have not used this `> ` format in other places in the spec. I would like to avoid introducing it and drawing unnecessary attention to this

```suggestion
Note that the above metadata structure has some known limitations:

* It cannot be used to describe claims in credentials that have the name `mandatory`, `value_type` or `display`.
* It is not possible to provide `mandatory`, `value_type` or `display` values for objects that contain claims 
* The `order` parameter cannot be used for claims within objects
* Arrays of unknown size cannot be described

These limitations are expected to be resolved in the second Implementer's Draft, a proposal can be viewed in [Issue 266](https://github.com/openid/OpenID4VCI/issues/266).

```
